demos/interactive_face_detection_demo_gapi/README.md

```diff
@@ -0,0 +1,97 @@
+
+## Running
+
+Running the application with the `-h` option yields the following usage message:
+
+```sh
+./interactive_face_detection_demo_gapi -h
+
+interactive_face_detection_demo [OPTION]
+Options:
+
+    -h                         Print a usage message
+    -i "<path>"                Required. Path to a video file (specify "cam" to work with camera).
+    -o "<path>"                Optional. Path to an output video file.
+    -m "<path>"                Required. Path to an .xml file with a trained Face Detection model.
+    -m_ag "<path>"             Optional. Path to an .xml file with a trained Age/Gender Recognition model.
+    -m_hp "<path>"             Optional. Path to an .xml file with a trained Head Pose Estimation model.
+    -m_em "<path>"             Optional. Path to an .xml file with a trained Emotions Recognition model.
+    -m_lm "<path>"             Optional. Path to an .xml file with a trained Facial Landmarks Estimation model.
+    -d "<device>"              Optional. Target device for Face Detection network (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. The demo will look for a suitable plugin for a specified device.
+    -d_ag "<device>"           Optional. Target device for Age/Gender Recognition network (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. The demo will look for a suitable plugin for a specified device.
+    -d_hp "<device>"           Optional. Target device for Head Pose Estimation network (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. The demo will look for a suitable plugin for a specified device.
+    -d_em "<device>"           Optional. Target device for Emotions Recognition network (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. The demo will look for a suitable plugin for a specified device.
+    -d_lm "<device>"           Optional. Target device for Facial Landmarks Estimation network (the list of available devices is shown below). Default value is CPU. Use "-d HETERO:<comma-separated_devices_list>" format to specify HETERO plugin. The demo will look for a suitable plugin for a specified device.
+    -no_show                   Optional. Do not show processed video.
+    -pc                        Option \"-pc\" is not supported in this version of the demo. Optional. Enable per-layer performance report
```

Comment:
Looks like the README file is still out of date - this option doesn't seem to exist anymore.

Reply:
Removed